components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -1267,7 +1267,7 @@ function Invoke-SetupEnvironmentWrapper {
     # the real environment, except for PATH; for that, push the
     # runtime path onto the front of the system path
     foreach($k in $env["Runtime"].keys) {
```

Comment:
Is it significant that this "Runtime", while above we have "RunTime" (same with "Buildtime" below)?

Reply:
No. Powershell hash keys and variables are not case sensitive.

Reply:
Hrmm... maybe it'd be better to be consistent.

If nothing else, it'd stop me from asking these questions :wink: